tests/__tests__/console/ConsoleTable_test.re

```diff
@@ -0,0 +1,428 @@
+        ~columns=[{width: 7}, {width: 4}],
+        ~children=[
+          Row.createElement(~children=["test a long column", "this"], ()),
+          Row.createElement(~children=["a", "b"], ()),
+        ],
+        (),
+      );
+    let expected =
+      ""
+      ++ "------------------\n"
+      ++ "| test a  | this |\n"
+      ++ "| long    |      |\n"
+      ++ "| column  |      |\n"
+      ++ "|---------+------|\n"
+      ++ "| a       | b    |\n"
+      ++ "------------------";
+    expect.string(actual).toEqual(expected);
+  });
+
+  test("Basic table JSX", ({expect}) => {
+    let actual =
+      <Table columns=[<ColumnConfig width=7 />, <ColumnConfig width=4 />]>
+        <Row> "testing" "1234" </Row>
+        <Row> "a" "b" </Row>
+      </Table>;
+    let expected =
+      ""
+      ++ "------------------\n"
+      ++ "| testing | 1234 |\n"
+      ++ "|---------+------|\n"
+      ++ "| a       | b    |\n"
+      ++ "------------------";
+    expect.string(actual).toEqual(expected);
+  });
+
+  test("Large table", ({expect}) => {
+    let actual =
+      <Table
+        columns=[
+          <ColumnConfig width=8 />,
+          <ColumnConfig width=16 />,
+          <ColumnConfig width=8 />,
+        ]>
+        <Row> "a1" "b1" "c1" </Row>
+        <Row> "a2" "b2 is a cell that needs to wrap" "c2" </Row>
+        <Row> "a3" "b3" "c3" "d3" </Row>
+        <Row> "a4" "b4" </Row>
+        <Row> "a5" "b5" "c5" </Row>
+      </Table>;
+    let expected =
+      ""
+      ++ "------------------------------------------\n"
+      ++ "| a1       | b1               | c1       |\n"
+      ++ "|----------+------------------+----------|\n"
+      ++ "| a2       | b2 is a cell     | c2       |\n"
+      ++ "|          | that needs to    |          |\n"
+      ++ "|          | wrap             |          |\n"
+      ++ "|----------+------------------+----------|\n"
+      ++ "| a3       | b3               | c3       |\n"
+      ++ "|----------+------------------+----------|\n"
+      ++ "| a4       | b4               |          |\n"
+      ++ "|----------+------------------+----------|\n"
+      ++ "| a5       | b5               | c5       |\n"
+      ++ "------------------------------------------";
+    expect.string(actual).toEqual(expected);
+  });
+
+  test("Formatted Table", ({expect}) => {
+    let bold_cell = Pastel.(<Pastel bold=true> "b1" </Pastel>);
+    let wrapped_blue_cell =
+      Pastel.(<Pastel color=Blue> "b2 is a cell that needs to wrap" </Pastel>);
+    let actual =
+      <Table
+        columns=[
+          <ColumnConfig width=8 />,
+          <ColumnConfig width=16 />,
+          <ColumnConfig width=8 />,
+        ]>
+        <Row> "a1" bold_cell "c1" </Row>
+        <Row> "a2" wrapped_blue_cell "c2" </Row>
+        <Row> "a3" "b3" "c3" "d3" </Row>
+        <Row> "a4" "b4" </Row>
+        <Row> "a5" "b5" "c5" </Row>
+      </Table>;
+    /*
+       Can't use Terminal mode in the printer but HumanReadable mode in the
```

Comment:
Interested to see why this was a problem

Reply:
Ah I understand now, I think you want to be using Pastel.useMode throughout to enforce Pastel.parse behaving in a consistent manner (so don't make a separate pastel via Pastel.Make)